DEV-633: Docs collator (#550)
cloudposse/docs: scripts/README.md

# Usage for docs-collator

> Docs collation tools works for 2 types of projects:

Reviewer: ```suggestion
Document collation supports 2 types of projects:
```

- components - `cloudposse/terraform-aws-component` - components from single repo `cloudposse/terraform-aws-component` under `modules/` dir 
- modules - `cloudposse/terraform-*` - for all public (non archived) repos except `cloudposse/terraform-aws-component` that are prefixed with `terraform-`

Python is used for both tools so make sure to install python requirements.

> ### Install python requirements

Reviewer: ```suggestion
### Install Python Requirements
```

```bash
pip install -r scripts/docs-collator/requirements.txt
```

> ### Render docs components

Reviewer: ```suggestion
### Render Documentation for Components
```

```bash
./scripts/render-docs-for-components.sh
```

> ### Render docs for modules

Reviewer: ```suggestion
### Render Documentation for Modules
```

> Since this script uses Github API - access token should be set as env var.

Reviewer: ```suggestion
Since this script uses the Github API, the GitHub access token should be set as the `PUBLIC_REPO_ACCESS_TOKEN ` environment variable.
```
